lib/dom.dart

```diff
@@ -454,24 +449,24 @@ class Text extends Node {
   @override
   String get text => data;
   @override
-  set text(String value) {
+  set text(covariant String value) {
```

Comment:
Probably worth a comment to explain why this is covariant?

Reply:
Done.